data-model/developer-schema/annotation/0.2.0/schema/annotation-event.json

```diff
@@ -0,0 +1,55 @@
+{
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "$id": "https://schemas.dissco.tech/schemas/developer-schema/annotation/0.2.0/annotation-for-mas.json",
+  "description": "Object sent to annotation processing service",
+  "properties": {
+    "@type": {
+      "@type": {
+        "type": "string",
+        "description": "The type of the object, in this case ods:BatchMetaData",
+        "const": "ods:AnnotationEvent"
+      }
+    },
+    "ods:hasAnnotation": {
+      "type": "array",
+      "items": {
+        "$ref": "https://schemas.dissco.tech/schemas/developer-schemas/annotations/0.2.0/annotation-for-mas.json"
+      }
+    },
+    "ods:jobID": {
+      "type": "string",
+      "description": "Handle of the job record, if the annotation was produced by a Machine Annotation Service",
+      "pattern": "https:\/\/hdl\\.handle\\.net\/\\w+\/(.){3}-(.){3}-(.){3}"
+    },
+    "ods:hasBatchMetadata": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "@type": {
+            "type": "string",
+            "description": "The type of the object, in this case ods:BatchMetaData",
+            "const": "ods:BatchMetaData"
+          },
+          "ods:placeInBatch": {
```

Comment:
placeInBatchqueue be better?

Reply:
Don't think this is related to the position in the queue, @southeo can probably better answer this.

Reply:
There are two components of an AnnotationEvent sent from the MAS to the processing service: BatchMetadata and the Annotations. Both of these can be lists, however, so the processing service needs a way to figure out which Annotation is associated with which BatchMetadata. The field `placeInBatch` is the value that links these two fields. 

(The batch metadata are the terms and values used to determine the search queries for to create annotations)